test-it/karma/utilities/TestHelpers.VireoRunner.js

```diff
@@ -3,10 +3,14 @@
     window.testHelpers = window.testHelpers || {};
 
     var rebootAndLoadVia = function (vireo, viaAbsolutePath) {
-        expect(viaAbsolutePath).toBeNonEmptyString();
+        // Jasmine Matchers library is not always ready in beforeAll so use jasmine core functions
+        expect(typeof viaAbsolutePath).toBe('string');
+        expect(viaAbsolutePath).not.toBe('');
```

Comment:
That is a bug in the jasmine matchers library that provides all those extra function on top of jasmine's standard matchers. So far it only effects utility files like this, it has not been a problem in the test suites yet.